### PR TITLE
Add docker build push

### PR DESCRIPTION
**Pull request by magnowest**
Enabling auto push of the image of this service to dockerhub via Github Actions to the dreamUp.ai org.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,13 +5,57 @@ A service for managing users in dreamup
...
 ```shell
-./scripts/run
+nvm use
+npm install
+npm run build
+```
+
+### Run Locally
+
+This project uses `dotenv` to read environment files. Multi-environment setups are supported via the environment variable `APP_ENV`. On start, the server will load `.env.${APP_ENV}`. The `start` script and the `pipeline-service.yml` file both assume `APP_ENV=local`, so you will need to create a file in the root of the directory called `.env.local`. For most purposes, copying `.env.test` should be sufficient. The `.gitignore` contains a rule to ignore `.env*.local` files.
```
Comment (shawnrushefsky) [inline README.md]:
```suggestion
This project uses `dotenv` to read environment files. Multi-environment setups are supported via the environment variable `APP_ENV`. On start, the server will load `.env.${APP_ENV}`. The `start` script and the `user-service.yml` file both assume `APP_ENV=local`, so you will need to create a file in the root of the directory called `.env.local`. For most purposes, copying `.env.test` should be sufficient. The `.gitignore` contains a rule to ignore `.env*.local` files.
```

**Review by shawnrushefsky — CHANGES_REQUESTED** [review]
Copy/paste error

File: README.md (hunk 1 above)
Comment (magnowest) [reply README.md]: Fixed.

**Review by shawnrushefsky — APPROVED** [review]
Thanks!

**Pull request merged**